src/stringformat.c

```diff
@@ -44,6 +61,23 @@ gdip_string_format_init (GpStringFormat *result)
 	result->charRangeCount = 0;
 }
 
+void
+gdip_create_generic_stringformats ()
+{
+	gdip_string_format_init (&stringFormatDefault);
+
+	gdip_string_format_init (&stringFormatTypographic);
+	stringFormatTypographic.formatFlags = StringFormatFlagsNoFitBlackBox | StringFormatFlagsLineLimit | StringFormatFlagsNoClip;
+	stringFormatTypographic.trimming = StringTrimmingNone;
+}
+
+void
+gdip_delete_generic_stringformats ()
+{
+	gdip_string_format_clear (&stringFormatDefault);
+	gdip_string_format_clear (&stringFormatTypographic);
```

Comment:
I don't think this cleanup is necessary since the allocated fields are never set. Or maybe I am incorrectly assuming that these objects are immutable? Not sure if there's a test for that.